### DISCUSSION TITLE
CascadingParameter with DrawerService?

### DISCUSSION THREAD

**Discussion by michaelcsikos** [ant-design-blazor/ant-design-blazor#3434] [Q&A]
Is it possible to use a `CascadingParameter` with the `DrawerService`?

**Comment by michaelcsikos** [comment]
I can't see `ContentParams` in `DrawerOptions` in 0.15.5.

**Comment by ElderJames** [comment]
This is becaues the DrawerService open the drawer from `AntContainer`, we can't use CascadingParameter  to pass paramerters. Please try to use FeedbackComponent https://antblazor.com/en-US/components/drawer#components-drawer-demo-drawer_service
